linkatos/activities.py

```diff
@@ -9,71 +9,69 @@ def is_empty(events):
     return ((events is None) or (len(events) == 0))
 
 
-def is_url(url_cache):
-    return url_cache is not None
+def is_url(element):
+    return element is not None
```

Comment:
I think, it would be better to put all cache related functions in a dedicated `cache.py` file. `activities.py` feels unrelated to cache internal operations.